File: Python/0001.py
def sum_of_multiples(limit):
    total_sum = 0
    for i in range(1, limit):
        if i % 3 == 0 or i % 5 == 0:
            total_sum += i
    return total_sum

# Define the limit
limit = 1000

# Calculate the result
result = sum_of_multiples(limit)

# Print the result
print(f"The sum of all multiples of 3 or 5 below {limit} is: {result}")

File: Rust/0001.rs
fn sum_of_multiples(limit: u32) -> u32 {
    let mut total_sum = 0;
    for i in 1..limit {
        if i % 3 == 0 || i % 5 == 0 {
            total_sum += i;
        }
    }
    total_sum
}

fn main() {
    let limit = 1000;
    let result = sum_of_multiples(limit);
    println!("The sum of all multiples of 3 or 5 below {} is: {}", limit, result);
}
